text/1000-migrations.rst

```diff
@@ -254,8 +271,17 @@ While the migration block is active:
 * The ``DESCRIBE CURRENT MIGRATION AS JSON`` statement returns a complete
   description of the current migration: statements that have already been
   recorded to be part of the migration script as well as automatically
-  generated list of statements required to complete the migration.  See
-  the "DESCRIBE MIGRATION" section below for details.
+  generated proposal for the next DDL statement to include to advance
+  the migration.  See the "DESCRIBE MIGRATION" section below for details.
+
+* The ``ALTER CURRENT MIGRATION REJECT PROPOSED`` statement is used to
+  inform the server that the currently proposed statement returned by
+  a prior call to ``DESCRIBE CURRENT MIGRATION AS JSON`` is not acceptable
+  and that the next execution of ``DESCRIBE`` should return an alternative
+  proposal.  If the client rejected all proposed statements, the next
```

Comment:
How does CLI finds out this? `complete: false` and no `proposed`? Seems to be very hacky way to do that, I would made something more explicit.

Also is there a way to find out what needs to be done? I.e what current diff is. CLI should show user something in this case.

Reply:
The only valid representation of the diff is DDL, and we've already shown the user all possible variants of it.  I don't think dumping the internal representation is a good idea.

> `complete: false` and no `proposed`? Seems to be very hacky way to do that,

Why?  Looks fairly unambiguous to me.